packages/components/tests/dummy/app/templates/components/tooltip.hbs

```diff
@@ -0,0 +1,210 @@
+
+  <Shw::Text::H3>Basic examples</Shw::Text::H3>
+
+  <Shw::Text::H4>On its own</Shw::Text::H4>
+  <Shw::Flex as |SF|>
+    <SF.Item @label="With a FlightIcon as content">
+      <Hds::TooltipButton @text="Here is more information">
+        <FlightIcon @name="alert-circle" />
+      </Hds::TooltipButton>
+    </SF.Item>
+
+    <SF.Item @label="With text as content and using bold and em tags to format tooltip text">
+      <Hds::TooltipButton @text="<b>Hello</b> to <em>you</em>!">
+        Hover me
+      </Hds::TooltipButton>
+    </SF.Item>
+
+    <SF.Item @label="With an HTML time element as content">
+      <Hds::TooltipButton @text="2023-03-30T19:18:01.684Z">
+        <time datetime="2023-03-30T19:18:01.684Z">Apr 5, 2023</time>
+      </Hds::TooltipButton>
+    </SF.Item>
+
+    <SF.Item @label="With really long content in tooltip">
+      <Hds::TooltipButton
+        @text="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+      >
+        <FlightIcon @name="alert-circle" />
+      </Hds::TooltipButton>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H4>Used <em>incorrectly</em> to wrap a block-level element (button wraps div tag)</Shw::Text::H4>
+  <Shw::Text::Body>(results in invalid HTML structure)</Shw::Text::Body>
+  <Hds::TooltipButton @text="Invalid usage">
+    <div><FlightIcon @name="alert-circle" /></div>
+  </Hds::TooltipButton>
+
+  <Shw::Text::H4>Used next to text</Shw::Text::H4>
+  <p>
+    <strong>Lorem ipsum dolor sit amet consectetur</strong>
+    <Hds::TooltipButton @text="Hello!"><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+  </p>
+
+  <Shw::Text::H4>Used within a sentence.</Shw::Text::H4>
+  <p>
+    Lorem ipsum dolor sit amet consectetur
+    <Hds::TooltipButton @text="Hello!"><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+    adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+  </p>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>States</Shw::Text::H3>
+
+  <Shw::Text::Body>
+    The button which serves as the opener for the tooltip has no special styles of its own other than for the focus
+    state.
+  </Shw::Text::Body>
+
+  <Shw::Flex as |SF|>
+    {{#let (array "default" "hover" "focus") as |states|}}
+      {{#each states as |state|}}
+        <SF.Item @label={{state}}>
+          <div class="shw-component-side-nav-home-link-wrapper">
+            <Hds::TooltipButton @text="Hello!" mock-state-value={{state}}><FlightIcon
+                @name="alert-circle"
+              /></Hds::TooltipButton>
+          </div>
+
+          {{#if (eq state "hover")}}
```

Comment:
Note: I don't think the code in this if block is actually a good idea but it's the only thing I could think of to sort of mock the hover state...